waku/v2/node/config.nim

```diff
@@ -458,11 +459,13 @@ proc parseCmdArg*(T: type Port, p: TaintedString): T =
 proc completeCmdArg*(T: type Port, val: TaintedString): seq[string] =
   return @[]
 
-func defaultListenAddress*(conf: WakuNodeConf): ValidIpAddress =
+proc defaultListenAddress*(): ValidIpAddress =
   # TODO: How should we select between IPv4 and IPv6
   # Maybe there should be a config option for this.
   (static ValidIpAddress.init("0.0.0.0"))
 
+proc defaultPrivateKey*(): PrivateKey =
```

Comment:
```suggestion
func defaultPrivateKey*(): PrivateKey =
```
Not sure if the generation of the key has side effects!

Reply:
Yes, it has side effects. That's why I changed all to `proc`: looking for consistency.